test/integration/exasol/test_exasol.py

```diff
@@ -53,11 +57,25 @@ class KeywordTest(fixtures.TablesTest):
 
     def test_keywords(self):
         keywords = config.db.execute(
-            "select distinct(lower(keyword)) as keyword "
-            + "from SYS.EXA_SQL_KEYWORDS where reserved = True order by keyword"
+            sql.text(
```

Comment:
Do we want to define some of the sql.text usage by defining Table objects? Was thinking maybe in the source code that could be nicer.

Worthwhile to add a refactoring tests tasks to do after the migration, during some engineering week?